packages/dapp-wallet/ui/src/components/NavMenu.jsx

```diff
@@ -1,13 +1,13 @@
 import { useMemo, forwardRef } from 'react';
 import { Link, useRouteMatch } from 'react-router-dom';
-import { makeStyles, useTheme } from '@material-ui/core/styles';
-import List from '@material-ui/core/List';
-import ListItem from '@material-ui/core/ListItem';
-import { ListItemText, ListItemIcon } from '@material-ui/core';
-import DashboardIcon from '@material-ui/icons/Dashboard';
-import Apps from '@material-ui/icons/Apps';
-import People from '@material-ui/icons/People';
-import AddCircle from '@material-ui/icons/AddCircle';
+import { makeStyles, useTheme } from '@mui/styles';
+import List from '@mui/material/List';
+import ListItem from '@mui/material/ListItem';
+import { ListItemText, ListItemIcon } from '@mui/material';
```

Comment:
I wonder if there's a more specific import for these `ListItem*` things than `'@mui/material'`.

Reply:
Good catch, fixed